frontend/src/bundles/studio/pages/studio.tsx

```diff
@@ -39,8 +48,40 @@ const Studio: React.FC = () => {
     }, [dispatch]);
 
     const handleSubmit = useCallback(() => {
-        navigate(AppRoute.ROOT);
-    }, [navigate]);
+        // TODO: REPLACE LOGIC WITH MULTIPLE SCENES
+        const scene = scenes[0];
+        const script = scripts[0];
+
+        if (!scene?.avatar && !script) {
```

Comment:
```suggestion
        if (!scene?.avatar || !script) {
```
Because if we have one script but no scene avatar we cannot create a video and vice versa